### PR TITLE
Updates in the pt-BR translation

### PR DESCRIPTION
**Pull request by felubra**
I've went further and translated the descriptions of the `ava --help` (CLI section) without leaving the original ones. Not sure if this is bad idea, please review.

### CODE REVIEW

**Comment by forresst** [comment]
I advise you to leave the version in English and to add in brackets the translation into Brazilian. This allows for those who have difficulties in English to have a translation when they execute the CLI. Here is the example of the French translation.

NOTE: I could not read your translation, I do not speak Brazilian. Do you know anyone who can do this review?

```console
$ ava --help

  Usage
    ava [<file|directory|glob> ...]

  Options
    --watch, -w             Re-run tests when tests and source files change (Re-exécute les tests quand les tests et les fichiers sources ont changé)
    --match, -m             Only run tests with matching title (Can be repeated) (Exécute seulement les tests qui correspondent au titre (peut être répété))
    --update-snapshots, -u  Update all snapshots (Mettre à jour les snapshots)
    --fail-fast             Stop after first test failure (Arrêter dès qu'un test échoue)
    --timeout, -T           Set global timeout (Définir un timeout global)
    --serial, -s            Run tests serially (Lancer les tests en série)
    --concurrency, -c       Max number of test files running at the same time (Default: CPU cores) (Nombre max de fichiers de test exécutés en même temps (Par défaut : coeurs CPU))
    --verbose, -v           Enable verbose output (Activer le mode verbose)
    --tap, -t               Generate TAP output (Générer une sortie au format TAP)
    --color                 Force color output (Force l'affichage avec des couleurs)
    --no-color              Disable color output (Désactiver l'affichage avec des couleurs)

  Examples (Exemples)
    ava
    ava test.js test2.js
    ava test-*.js
    ava test

  Default patterns when no arguments (Les patterns par défaut quand il n'y a pas d'arguments) :
  test.js test-*.js test/**/*.js **/__tests__/**/*.js **/*.test.js
```

**Comment by felubra** [comment]
I've restored the original descriptions and put them alongside with the translations.
I will try to find more people to do the review, thanks!

File: pt_BR/readme.md [hunk 1]
```diff
@@ -137,35 +137,36 @@ AVA vem com um modo inteligente de observação. [Saiba mais na receita](docs/re
...
-    --timeout, -T      Define um timeout global
-    --concurrency, -c  Número máximo de arquivos-teste rodando ao mesmo tempo (EXPERIMENTAL)
-
-  Exemplos
+  Usage (Uso)
+    ava [<file|directory|glob> ...]
+
+  Options (Opções)
+    --init             Add AVA to your project (Adiciona o AVA ao seu projeto)
+    --fail-fast        Stop after first test failure (Para depois da primeira falha em um teste)
```
Comment (fabiocharpinel) [inline pt_BR/readme.md]: Sugiro colocar sempre o verbo no infinitivo ou no imperativo. Exemplo:

PARAR depois da primeira falha no teste ou PARE depois da primeira falha no teste.

RODAR os testes em série ou RODE os testes em série.

File: pt_BR/readme.md [hunk 2]
```diff
@@ -137,35 +137,36 @@ AVA vem com um modo inteligente de observação. [Saiba mais na receita](docs/re
...
+    --serial, -s       Run tests serially (Roda os testes em série)
+    --require, -r      Module to preload (Can be repeated) (Módulo para pré-carregar (pode ser repetido))
+    --tap, -t          Generate TAP output (Gera uma saída TAP)
+    --verbose, -v      Enable verbose output (Habilita a saída verborrágica)
+    --no-cache         Disable the transpiler cache (Desabilita o cache do transpiler)
+    --no-power-assert  Disable Power Assert (Desabilita o Power Assert)
+    --match, -m        Only run tests with matching title (Can be repeated) (Somente roda testes que casem com o título informado (pode ser repetido))
```
Comment (fabiocharpinel) [inline pt_BR/readme.md]: Apenas rode testes com o mesmo título informado.

File: pt_BR/readme.md [hunk 3]
```diff
@@ -137,35 +137,36 @@ AVA vem com um modo inteligente de observação. [Saiba mais na receita](docs/re
...
+    --watch, -w        Re-run tests when tests and source files change (Re-roda os testes quando os testes e os arquivos código-fonte mudarem)
```
Comment (fabiocharpinel) [inline pt_BR/readme.md]: Refaça os testes quando o código-fonte mudar.

File: pt_BR/readme.md [hunk 4]
```diff
@@ -137,35 +137,36 @@ AVA vem com um modo inteligente de observação. [Saiba mais na receita](docs/re
...
+  Examples (Exemplos)
     ava
     ava test.js test2.js
     ava test-*.js
     ava test
     ava --init
     ava --init foo.js
 
-  Padrões de arquivo pré-definidos, quando executado sem argumetos:
+  Default patterns when no arguments (Padrões pré-definidos, quando sem argumentos):
```
Comment (fabiocharpinel) [inline pt_BR/readme.md]: Padrões pré-definidos quando não há argumentos.

**Comment by fabiocharpinel** [comment]
@felubra, I submitted some changes that I found necessary. I hope it helps.

File: pt_BR/readme.md (hunk 1 above)
Comment (felubra) [reply pt_BR/readme.md]: Boa idéia dos tempos verbais. Vou padronizar para o imperativo.

File: pt_BR/readme.md (hunk 3 above)
Comment (felubra) [reply pt_BR/readme.md]: 'Refaça' é bem melhor do que 're-rode', valeu.

**Pull request closed without merging**